src/bin/commands/build-docs.ts

```diff
@@ -0,0 +1,24 @@
+import { exec as execSync } from 'child_process';
+import ora from 'ora-classic';
```

Comment:
You may want to switch to @favware/colorette-spinner which has both ESM and CJS bundles coz not st00pid
(also it uses colorette instead of chalk)